Plugins/NativeCamera/Source/Apple/CameraDevice.mm

```diff
@@ -666,11 +720,11 @@ fragment float4 fragmentShader(RasterizerData in [[stage_in]],
 
                     [m_impl->currentCommandBuffer addCompletedHandler:^(id<MTLCommandBuffer>) {
                         if (textureY != nil) {
-                            [textureY setPurgeableState:MTLPurgeableStateEmpty];
+                            [textureY setPurgeableState:MTLPurgeableStateVolatile];
                         }
 
                         if (textureCbCr != nil) {
-                            [textureCbCr setPurgeableState:MTLPurgeableStateEmpty];
+                            [textureCbCr setPurgeableState:MTLPurgeableStateVolatile];
```

Comment:
It might be good to get @ryantrem's input on this.

Reply:
Dunno, the error message Cedric included in the PR description combined with the Apple docs are very confusing...

Reply:
I got the validation error with Volatile as well, som definitely not a valid fix.